ccd-adapter/build.gradle

```diff
@@ -0,0 +1,14 @@
+group 'uk.gov.hmcts.cmc'
+version '1.1.0'
```

Comment:
we should be able to extract the version somehow, I assume this would never be different to the parent?

Reply:
It's definitely possible to use parent's version, I'm quite confident we've already done that.

Reply:
Removed as were added by idea. Defaults would be from parent gradle.